docs/src/main/tut/pull_request.md

```diff
@@ -100,9 +99,10 @@ createPullRequestData.exec[cats.Id, HttpResponse[String]]() match {
 }
 ```
 
-On the other hand, we can pass a `issue` id (through `NewPullRequestIssue` object) instead of the title and the body to get this parameter of the issue
+On the other hand, we can pass a `issue` id (through `NewPullRequestIssue` object)
+instead of the title and the body to get this parameter of the issue.
```

Comment:
On the other hand, we can pass an `issue` id (through a `NewPullRequestIssue` object) instead of the title and body.